### PR TITLE
CHAAAOOOOSSSSS DDUUUUUNNNKKK

### PR DESCRIPTION
**Pull request by McDonald072**
insert a nuke core into a basketball to perform the dunk of the millennium

suggestions welcome

### CODE REVIEW

**Comment by Cruix** [comment]
I kind of want to see this happen

**Comment by matskuman5** [comment]
>explosion(get_turf(user), 14, 28, 56, 112)

glorious

**Comment by McDonald072** [comment]
I think it should be even bigger

**Comment by ShadowDeath6** [comment]
Set the color of the bball to be green or something pls
:+1:

**Comment by Amelia0010** [comment]
But anyone who cares even slightly about greentext would never do this. 
You're gibbed when you do it, which means it's impossible to greentext your "steal the nuke core" goal. And the only people who actually get the tools to get the nuke core are the only ones who can't ~~sacrifice~~ blow it up it without redtext. 

It would be bloody hilarious, but only a select few people would ever do it

**Comment by ArgyleSweatshop** [comment]
+1

The alarm sound needs to be replaced with an actual space jam midi.

**Comment by ArgyleSweatshop** [comment]
Also there is no way to get a nuke core object. It's either in the nuke or in a nuke core container. You need to add a way to remove it from either the core container or a way to remove the core from the nuke without a container.

**Comment by McDonald072** [comment]
Huh, I thought you could remove it without the box.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Hm, but now we have a mobile source of lethal radiation. Any suggestions on how to handle this better?

**Comment by ArgyleSweatshop** [comment]
yeah but it will also kill the person carrying. It's fine.

McDonald, sneak in a code change that makes it so you dont need nuke screwdriver to open the panel at start

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
\>.>
Not if that person has a radsuit. 
(I'm McDonald by the way)

**Comment by ArgyleSweatshop** [comment]
Don't worry about it. Situations where that might arise are minimal

**Comment by Slinky5890** [comment]
Wait, mcdonald is useful?? When did this happen?

**Comment by Dagdammit** [comment]
Here's what you do.

If someone has objective to steal nuke core, there is a chance (25%?) that their final escape/hijack/die objective will be replaced by "Use the stolen nuclear core to create the Ultimate B-Ball and perform a Chaos Dunk." http://hoopz-barkley-saga.wikia.com/wiki/Ultimate_B-Ball

The Ultimate B-ball is created by using a sealed nuke core container on a regular basketball. The Ultimate B-Ball, when used in hand, either informs you that this location is unworthy is unworthy of such a magnificent ball or gives a yes/no popup reading "Dare you use the power of the ultimate b-ball?"

Choosing "yes" will have consequences comparable to placing a gang dominator, with a handful of differences:
-The placed item is an intangible, indestructible Spirit Basketball Hoop with a semitransparent sprite. 
-The station warning speaks of "Catastrophic b-ball energies gathering in <location>. The ULTIMATE B-BALL is in play. All personnel must prevent its use by any means necessary."
-The Ultimate B-Ball becomes a nodrop item which the user cannot stop dribbling. They must survive for 60 (?) seconds of uninterrupted dribbling, amassing power. (This may be a powered-up state; the ball could bestow a passive stimpack effect and demolish walls a la sonic jackhammer)
-Once charged, using Ultimate B-Ball on the Spirit Hoop will fulfill the objective and either detonate a maxcap or spawn a singularity at that location.

**Comment by ShadowDeath6** [comment]
Make this only work with real basketballs, so people can't abuse the holodeck to delete the nuke core

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
You can't reliably get a basketball anywhere else but the holodeck. Perhaps we should add basketballs as maint loot?

**Comment by ShadowDeath6** [comment]
There's a secret basketball in the bartender's wall

**Comment by Dagdammit** [comment]
Doesn't it just create sonething real when you use holodeck items to make something?

**Comment by McDonald072** [comment]
It doesn't always appear.

Dag, I'm not really sure how that is relevant, can you expand on it?

**Comment by Kmc2000** [comment]
holy shit yes

**Comment by ArgyleSweatshop** [comment]
Now just make regular screwdriver able to open core so we can defeat nuke ops with a basketball

**Comment by McDonald072** [comment]
no

**Comment by ShadowDeath6** [comment]
It does always appear actually

**Comment by McDonald072** [comment]
How come the last 5 times I checked it wasn't there

**Comment by ShadowDeath6** [comment]
I'm pretty sure it's always there, I'll check when I get home

File: code/game/gamemodes/objective_items.dm [hunk 1]
```diff
@@ -80,10 +80,16 @@
 	name = "the heavily radioactive plutonium core from the onboard self-destruct. Take care to wear the proper safety equipment when extracting the core"
 	targetitem = /obj/item/nuke_core
 	difficulty = 15
+	altitems = list(/obj/item/toy/beach_ball/holoball/chaos)
```
Comment (ShadowDeath6) [inline code/game/gamemodes/objective_items.dm:83]: Where's this used

Comment (McDonald072) [reply code/game/gamemodes/objective_items.dm:83]: [here](https://github.com/yogstation13/yogstation/blob/f21ef0459052b2418969ad41989615ea4c74005b/code/game/gamemodes/objective.dm#L544)

**Comment by Dagdammit** [comment]
Wait, so do you need a nonstandard b-ball item to contain nuke core in the first place? I thought it was adding the nuke core/nuke core container that made a regular b-ball into an ultimate one.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
You use a standard, non holodeck (since you can't get those to the vault anyway) basketball on the self-destruct terminal in the vault. It will turn into a chaos basketball with which you can dunk.

**Comment by ShadowDeath6** [comment]
you were right, basketball doesn't spawn 100% of the time

**Comment by Dagdammit** [comment]
Can you order them via cargo?

McDonald, the thought in my head (and your call on whether this is any
better) was that you had to get the nuke core into the standard nuke core
objective container, wait for it to seal itself, and THEN use THAT on a
regular basketball.

That, plus the 60 seconds of dribbling to build charge for the chaos dunk
(and having to trigger it at the same location the warning message told the
crew about), was intended to give the crew a shot at preventing the chaos
dunk after the nuke core's been stolen?

Sounds like in the present version the chaos dunk isn't an added layer of
challenge, but instead something that makes the nuke core theft objective
way easier? I can just activate the chaos dunk in the vault as soon as I've
removed the core from the self destruct, and win so long as I stay alive
for 10 whole seconds.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
The thing is, doing a chaos dunk is not an objective. You are sacrificing your life and greentext for a stupid meme. 

I'll consider making it preventable though, as it causes a huge amount of damage.

**Comment by ShadowDeath6** [comment]
I think the nuke should be accessible with any screwdriver so traitors with grorious deathu can use it to dunk a quarter of the station

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Nah, it was PRd by Argyle recently and the feedback was universally negative

**Comment by ShadowDeath6** [comment]
I actually got a better idea for the screwdriver thing and I'll add it if this is merged

**Comment by Dagdammit** [comment]
Ohhh, so it's still memetext over greentext. Gotcha.

**Comment by ShadowDeath6** [comment]
I think you fucked up your latest commit
It also might not help that you're pushing from master

**Comment by McDonald072** [comment]
The latest commit comments out the bit that allows you to create a chaos ball, what did I fuck up?

**Comment by ShadowDeath6** [comment]
It makes it so that you can't create a chaos bball

**Comment by McDonald072** [comment]
Well yeah that's the point. It's admin-spawn only for now so it's merge ready.

**Comment by Ephemeralis** [comment]
This is hilariously bad in the best possible way.

**Comment by Mrcreepssss** [comment]
maybe traitors getting steal the nuke core objective also getting glorious death should be increased? greentext+memetext=besttext

**Pull request merged**